StarryExpanse/Structs/InteractabilityPollResponse.cpp

```diff
@@ -0,0 +1,16 @@
+//
+// Copyright 2017 59 Volt Entertainment, all rights reserved.
+//
+
+#include "InteractabilityPollResponse.h"
+
+FInteractabilityPollResponse FInteractabilityPollResponse::BasicResponse() {
```

Comment:
What about having the "Basic" response be the default constructed object? We can put those four values into the header files as defaults, and then delete this method and have the callers just return a response made with the default constructor.

Reply:
I'm just wary about what happens if we decide we want to split "Basic" into two different templates, and we're going to have to search and replace in a lot of places but it's going to be hard to do because we don't call any methods, it's just implicit. I prefer explicitness.

Reply:
I think it'll be easy to find those places in two different ways:

1. Use the code browser in Visual Studio. I think it'll show all places where that type is instantiated.
2. Manually delete the constructor and compile. It'll fail at each use.

As it stands a FInteractabilityPollResponse can still be instantiated with the default constructor in addition to calling ::Basic().